db/schema.rb

```diff
@@ -16,7 +16,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "cart_items", force: true do |t|
+  create_table "cart_items", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.